src/kava.js

```diff
@@ -78,7 +78,8 @@ class Kava extends BasePlugin {
       eventIndex: 1,
       bufferTime: 0.0,
       bufferTimeSum: 0.0,
-      playTimeSum: 0.0
+      playTimeSum: 0.0,
+      userId: this.config.userId
```

Comment:
this is not required as it will be queried in every event when getEventModel is called (where you added the new param) https://github.com/kaltura/playkit-js-kava/blob/4b414b11b0bf496c44770705b18556eccc0fcadb/src/kava-event-model.js#L268

Reply:
It requested to send it in every event

Reply:
see how it is done in `_setModelDelegates`